website/docs/security/cve/enterprise/GHSA-378h-hpjp-gxfc.md

```diff
@@ -0,0 +1,39 @@
+
+# Malicious links can be crafted by users and shown in the UI
+
+## Impact
+Unpatched versions of Weave GitOps Enterprise are vulnerable to a cross-site scripting (XSS) bug allowing a malicious user to inject a javascript: link in the UI. When clicked by a victim user, the script will execute with the victim's permission.
+
+The exposure appears in Weave GitOps Enterprise UI via a GitopsCluster dashboard link. An annotation can be added to a GitopsCluster custom resource.
+
+```yaml
+apiVersion: gitops.weave.works/v1alpha1
+kind: GitopsCluster
+metadata:
+  name: demo-02
+  namespace: default
+  annotations:
+    metadata.weave.works/dashboard.hellothere: "javascript:alert('hello there ' + window.localStorage.getItem('name'));"
+```
+
+The attacker does not need access to the Weave GitOps UI to craft an attack, access to a Git repository that contains resources that are displayed is all that is needed.
```

Comment:
This should also mention modifying K8s resources per https://github.com/weaveworks/weave-gitops/pull/2621/files/23fef237f72a9725ec84ae3a24a14d919b9da9f2#diff-e35735e4a39b31caa23ea95bcf1cb292668e76a8922663baddca716c67382032R30

Reply:
added https://github.com/weaveworks/weave-gitops/pull/2621/commits/d84e0c1c09f18b439bc5f2dbc83265e2b4dab076